sentry-android-ndk/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest package="io.sentry.android.ndk" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="io.sentry.android.ndk">
+  <application>
+    <meta-data android:name="io.sentry.ndk" android:value="true"/>
```

Comment:
When the manifest merging happens, I assume what's set in the apps manifest will win? We wouldn't want to force init even when the user is trying to turn it off (even if the ndk package is installed)

Reply:
@bruno-garcia yes, there are merging rules, App's manifest is a high priority, but needs to set the merging rule, unfortunately, added an example and let's discuss it.
another way would be via `buildConfigField` which is gonna be available on generated BuildConfig class.
https://developer.android.com/studio/build/gradle-tips

Reply:
Wouldn't it be simpler to just not add this configuration here? Leave it `true` in the code instead?

Reply:
I second @bruno-garcia comments. Just leave it enabled in the code and let apps use a `meta-data` to turn it off.

Reply:
or that, done :D